homeassistant/components/airzone_cloud/select.py

```diff
@@ -0,0 +1,129 @@
+"""Support for the Airzone Cloud select."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Any, Final
+
+from aioairzone_cloud.common import AirQualityMode
+from aioairzone_cloud.const import (
+    API_AQ_MODE_CONF,
+    API_VALUE,
+    AZD_AQ_MODE_CONF,
+    AZD_ZONES,
+)
+
+from homeassistant.components.select import SelectEntity, SelectEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import AirzoneUpdateCoordinator
+from .entity import AirzoneEntity, AirzoneZoneEntity
+
+
+@dataclass(frozen=True)
+class AirzoneSelectDescriptionMixin:
+    """Define an entity description mixin for select entities."""
+
+    api_param: str
+    options_dict: dict[str, str]
+
+
+@dataclass(frozen=True)
+class AirzoneSelectDescription(SelectEntityDescription, AirzoneSelectDescriptionMixin):
+    """Class to describe an Airzone select entity."""
+
+
+AIR_QUALITY_DICT: Final[dict[str, str]] = {
```

Comment:
```suggestion
AIR_QUALITY_MAP: Final[dict[str, str]] = {
```

Reply:
Done in 1ee48f0c4f9b892b825ef4b43441bc1d4edded53